### PR TITLE
Minor CI cleanup

### PR DESCRIPTION
**Pull request by Nyholm**
This will speed up the test a little

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Can we use composer 2 instead of flex?

**Comment by GrahamCampbell** [comment]
Just like https://github.com/laravel/framework/pull/33158.

**Comment by Nyholm** [comment]
Yes, of course. I was not aware of that setting.

**Comment by GrahamCampbell** [comment]
> I was not aware of that setting.

It's quite new. I asked them to add it. :)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,13 +13,12 @@ jobs:
 
     steps:
       - name: Set up PHP
-        uses: shivammathur/setup-php@1.7.0
+        uses: shivammathur/setup-php@2.3.1
```
Comment (GrahamCampbell) [inline .github/workflows/ci.yml:16]: Could we make this just `@v2` like we did for checkout and node? I don't think we need to pin?

**Comment by gmponos** [comment]
Thank you

**Comment by Nyholm** [comment]
Great. Thank you for the reviews

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Nyholm) [reply .github/workflows/ci.yml:16]: Actually, that is not the same.. 

Github actions maintainers are suggested to update the tag. 

So `shivammathur/setup-php@2.3.1` means "get version 2.3.1. 
And `shivammathur/setup-php@v2` means "get version v2. 

It is nothing like composer. If maintainers of `shivammathur/setup-php` does not update (move) the tag, then we are stuck with an old version... Just like now.. 

With this we know what version we are using. If a future version includes stuff that we like, then we just change the version here.

Comment (GrahamCampbell) [reply .github/workflows/ci.yml:16]: v2 does actually seem to get 2.3.1, otherwise the composer:v2 thing wouldn't work.

**Pull request merged**